indico/modules/search/client/js/components/results/File.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {List, Icon} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import './File.module.scss';
+import moment from 'moment';
+
+const iconSelector = type => {
+  const size = {size: 'large'};
+  switch (type) {
+    case 'file-word':
+      return {color: 'blue', ...size, name: 'file word outline'};
```

Comment:
I would call `size` something like `attributes` or `otherAttributes`. I would also move it to the end (before the `}`) for readability reasons.